CHANGELOG.md

```diff
@@ -13,7 +13,10 @@
 - PR #1040: Update version of numba in development conda yaml files
 - PR #1043: Updates to accomodate cuDF python code reorganization
 - PR #1044 Remove nvidia driver installation from ci/cpu/build.sh
```

Comment:
its a super duper small thing, this line is missing a `:`

Reply:
Good catch!